Resources/Prototypes/Nyanotrasen/Entities/Mobs/NPCs/fungi.yml

```diff
@@ -0,0 +1,79 @@
+- type: entity
+  parent: SimpleSpaceMobBase
+  id: MobFungusBase
+  name: fungus
+  abstract: true
+  components:
+  - type: Bloodstream
+    bloodReagent: Mold
+  - type: PsionicInsulation
+
+- type: entity
+  parent: MobFungusBase
+  id: MobFungusGasSpore
+  name: gas spore
+  description: A floating ball of gas.
+  components:
+  - type: CombatMode
+    combatToggleAction:
+      enabled: false
+      autoPopulate: false
+      name: action-name-combat
```

Comment:
Is CombatMode necessary if it has no means of attack?

Reply:
this is making it so they don't have combat mode despite inheriting it. See mice